Project.toml

```
@@ -1,7 +1,7 @@
 name = "StatsBase"
 uuid = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 authors = ["JuliaStats"]
-version = "0.34.4"
+version = "0.35.0"
```

We should really go to 1.0 next time we make a breaking release.
```suggestion
version = "1.0"
```

It would be good to check whether we want to make other breaking changes though (I don't have ideas off the top of my head).